sdcm/cluster.py

```diff
@@ -1286,7 +1287,7 @@ def upgrade_mgmt(self, scylla_mgmt_repo):
         if self.is_rhel_like():
             self.remoter.run('sudo yum update scylla-manager -y')
         else:
-            self.remoter.run('sudo apt-get update')
+            self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
```

Comment:
After update apt source url, and rename jessie-stable, the `apt-get update` will still succeed.
So the ignore_status=True isn't necessary.

Reply:
the apt-get always fail on something on its list in this case.
for example:
``` 16:28:13 [3.82.188.131] [stdout] Err http://cloudfront.debian.net jessie-backports/main Sources
16:28:13 [3.82.188.131] [stdout]   404  Not Found [IP: 13.249.44.66 80]
16:28:13 [3.82.188.131] [stdout] Err http://cloudfront.debian.net jessie-backports/main amd64 Packages
16:28:13 [3.82.188.131] [stdout]   404  Not Found [IP: 13.249.44.66 80]
16:28:13 [3.82.188.131] [stdout] Ign http://cloudfront.debian.net jessie-backports/main Translation-en_US
16:28:13 [3.82.188.131] [stdout] Ign http://cloudfront.debian.net jessie-backports/main Translation-en
16:28:16 [3.82.188.131] [stdout] Fetched 13.5 MB in 4s (3,146 kB/s)
16:28:16 [3.82.188.131] [stderr] W: Failed to fetch http://cloudfront.debian.net/debian/dists/jessie-backports/main/source/Sources  404  Not Found [IP: 13.249.44.66 80]
16:28:16 [3.82.188.131] [stderr] 
16:28:16 [3.82.188.131] [stderr] W: Failed to fetch http://cloudfront.debian.net/debian/dists/jessie-backports/main/binary-amd64/Packages  404  Not Found [IP: 13.249.44.66 80]
16:28:16 [3.82.188.131] [stderr] 
16:28:16 [3.82.188.131] [stderr] E: Some index files failed to download. They have been ignored, or old ones used instead. ```
so it must keep the 'ignore_status'

Reply:
This is the main issue we want to workaround, check jessie-backport of all source files, not just jessie-backports.list

Reply:
Another overall workaround is trying to replace all jessie-backports source url to `archive.debian.org`

```
sudo sed -i -e 's/:\/\/.*\/debian jessie-backports /:\/\/archive.debian.org\/debian jessie-backports /g' /etc/apt/sources.list.d/*.list
```

Reply:
ping @yarongilor